packages/composition/objectstorage/cloudscale.yml

```diff
@@ -135,13 +127,6 @@ parameters:
                         rma: rma
                         lpg: lpg
 
-    additionalResources:
-      provider-cloudscale-secrets-objectsusers:
-        apiVersion: v1
-        kind: Namespace
-        metadata:
-          name: ${pkg.appcat.composition.objectstorage.cloudscale:secretNamespaces:objectsusers}
```

Comment:
I'm not sure if this should actually be removed or not?

Reply:
No this one is still necessary, it's the namespace where we save the intermediate secrets of the Managed Resources in the composition.

Reply:
ah yeah, that's what I thought. Removed it from the changes list again.